src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs

```diff
@@ -160,11 +160,13 @@ internal partial class CodeFixService : ICodeFixService
             ICodeActionRequestPriorityProvider priorityProvider,
             CodeActionOptionsProvider fallbackOptions,
             Func<string, IDisposable?> addOperationScope,
+            bool includeConfigureAndSuppressFixer,
             [EnumeratorCancellation] CancellationToken cancellationToken)
         {
+            // 'includeConfigureAndSuppressFixer' would be false when the method is invoked by non-vs LSP client.
             // We only need to compute suppression/configuration fixes when request priority is
             // 'CodeActionPriorityRequest.Lowest' or 'CodeActionPriorityRequest.None'.
-            var includeSuppressionFixes = priorityProvider.Priority is CodeActionRequestPriority.Lowest or CodeActionRequestPriority.None;
+            var includeSuppressionFixes = includeConfigureAndSuppressFixer && priorityProvider.Priority is CodeActionRequestPriority.Lowest or CodeActionRequestPriority.None;
```

Comment:
The core of the change.
I don't know if there is a better way to filter configure and suppression fixers.

Reply:
An alternative that might be less intrusive is to remove any code actions that inherit from https://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/CodeFixes/Suppression/NestedSuppressionCodeAction.cs,cbb0a6dbad07cb71 after you get the code actions back in the LSP handler.  

Might be better just because this is more of a temporary fix.

Reply:
It might not be enough to only filter out nested supression code action, because there are also configure code action.
Let me check if it is possible to do this in from the code action side

Reply:
OK, seems like we wrap all suppression code action in an no-op operation.
https://sourceroslyn.io/#Microsoft.CodeAnalysis.LanguageServer.Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs,325